### PR TITLE
Update Middleware categories

### PR DESCRIPTION
**Pull request by spadgett**
openshift web console 2017-09-06 08-55-50

### CODE REVIEW

File: dist/origin-web-catalogs.js [hunk 1]
```diff
@@ -293,25 +293,50 @@ webpackJsonp([ 0, 1 ], [ function(e, t) {
...
+                icon: "font-icon icon-jboss"
+            }, {
+                id: "processserver",
+                label: "BPM Suite",
+                tags: [ "processserver" ],
+                icon: "font-icon icon-jboss"
+            }, {
+                id: "decisionserver",
+                label: "BRMS",
+                tags: [ "processserver" ],
```
Comment (knrc) [inline dist/origin-web-catalogs.js]: decisionserver

**Comment by spadgett** [comment]
@rhamilto Need to decide if we want a max-width for long subcategory names

**Comment by spadgett** [comment]
@sspeiche @serenamarie125

File: dist/origin-web-catalogs.js [hunk 2]
```diff
@@ -293,25 +293,50 @@ webpackJsonp([ 0, 1 ], [ function(e, t) {
...
+                tags: [ "decisionserver" ],
+                icon: "font-icon icon-jboss"
+            }, {
+                id: "datagrid",
+                label: "Data Grid",
+                tags: [ "datagrid" ],
+                icon: "font-icon icon-jboss"
+            }, {
+                id: "datavirt",
+                label: "Data Virtualization",
```
Comment (spadgett) [inline dist/origin-web-catalogs.js]: @knrc Is there a shorter label that we could use here? "Data Virt" maybe? This makes the subcategory tab larger than others.

**Comment by knrc** [comment]
On 6 September 2017 at 06:57, Sam Padgett <notifications@github.com> wrote:

> @knrc <https://github.com/knrc> Is there a shorter label that we could
> use here? "Data Virt" maybe? This makes the subcategory tab larger than
> others.
>
I would have no objections to this, make the change and if someone objects
they can raise it as an issue :)

   Kev

**Pull request merged**